lib/addons/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "./src"
```

Comment:
I think this line is causing problems for some addons (addon-docs) in particular it seems:

```
lerna ERR! yarn run prepare exited 2 in '@storybook/addon-docs'
lerna ERR! yarn run prepare stdout:
$ node ../../scripts/prepare.js
src/preset.ts(50,56): error TS2345: Argument of type '(framework: string) => string | number' is not assignable to parameter of type 'FrameworkPresetMapper'.
  Type 'string | number' is not assignable to type 'string | void'.
    Type 'number' is not assignable to type 'string | void'.
```

Reply:
found a fix